src/django_htmx/management/commands/downloadhtmx.py

```diff
@@ -0,0 +1,77 @@
+from __future__ import annotations
```

Comment:
How about we call the management command `htmx` and have a subparser for the `download` subcommand, allowing the possibility of more in the future?

Reply:
I have never considered Django management subcommands as I thought it's not idiomatic, but it doesn't sound like a bad idea at all!

Reply:
Done in 5df5cbaa01e179cadf0e2ea8fee64be7435f06d4